### DISCUSSION TITLE
Why did my login stop working?

### DISCUSSION THREAD

**Discussion by KevinxFinnerty** [UltimaHoarder/UltimaScraper#1765] [Q&A]
Everything was fine last week but now I can't get in:

![image](https://user-images.githubusercontent.com/94399793/141841654-98cfc81a-248e-4092-b9ca-25793ed82248.png)

**Comment by medlhed420** [comment]
I'm having this problem too.

I went back into developer tools and got my got my latest cookie. The only value that has changes is "sess="

I updated the values but I am still getting auth failure.
